io/www/src/fragment/common.js

```diff
@@ -44,36 +44,126 @@ async function computeBody(response, context) {
     return body;
 }
 
-async function internalFetch(path, context) {
+function createTimeoutPromise(timeout) {
+    return new Promise((_, reject) => {
+        setTimeout(() => {
+            const error = new Error(`Request timed out after ${timeout}ms`);
```

Comment:
could you reformulate it a bit?
it is exactly like the message inside native fetch.js and thus one can't be sure from which code this message was logged:
<img width="396" height="109" alt="image" src="https://github.com/user-attachments/assets/898f5e2b-43b5-48c2-b4e0-c2859796be62" />